train.py

```diff
@@ -25,49 +68,60 @@ class TrainingOutput(BaseModel):
 
 def train(
     input_videos: Path = Input(
-        description="A zip file containing videos and matching .txt captions.",
+        description="A zip file containing videos that will be used for training. If you include captions, include them as one .txt file per video, e.g. my-video.mp4 should have a caption file named my-video.txt. If you don't include captions, you can use autocaptioning (enabled by default).",
+        default=None,
+    ),
+    trigger_word: str = Input(
+        description="The trigger word refers to the object, style or concept you are training on. Pick a string that isn't a real word, like TOK or something related to what's being trained, like STYLE3D. The trigger word you specify here will be associated with all videos during training. Then when you use your LoRA, you can include the trigger word in prompts to help activate the LoRA.",
+        default="TOK",
+    ),
```

Comment:
naive question - should we default to autocaptioning or default to not autocaptioning?

Reply:
ah jk I see that we only autocaption videos w/out captions. makes more sense now. might want to call that behavior out.